libraries/AP_BattMonitor/AP_BattMonitor_SMBus_Maxell.cpp

```diff
@@ -54,6 +64,11 @@ void AP_BattMonitor_SMBus_Maxell::read()
 
 void AP_BattMonitor_SMBus_Maxell::timer()
 {
+	// _pec_confirmed set true after confirming if it support PEC
+    if (!_pec_confirmed) {
+        _pec_confirmed = get_pec_support();
+    }
```

Comment:
If _pec_confirmed is false, the version is unknown.
I think that it can be determined that SMBUS is not supported if the version is unknown.